### PR TITLE
added sumset.py Fixes: #{6563}

### PR DESCRIPTION
**Pull request by happiestbee**
### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by CenTdemeern1** [comment]
Hey, the original author of the issue this fixes (#6563) said they'd like to do the implementation themselves. Please let them.

File: maths/sumset.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+"""
+
```
Comment (CaedenPH) [inline maths/sumset.py:2]:
```suggestion
```

File: maths/sumset.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+Calculates the SumSet of two sets of numbers (A and B)
+
+Source:
+    https://en.wikipedia.org/wiki/Sumset
+
```
Comment (CaedenPH) [inline maths/sumset.py:7]:
```suggestion
https://en.wikipedia.org/wiki/Sumset
```

File: maths/sumset.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+def sumset(set_a: set, set_b: set) -> set:
+    """
+    :param first set: a set of numbers
+    :param second set: a set of numbers
+    :return: the nth number in Sylvester's sequence
+
+    >>> sumset({1, 2, 3}, {4, 5, 6})
+    {5, 6, 7, 8, 9}
+
```
Comment (CaedenPH) [inline maths/sumset.py:19]:
```suggestion
```

File: maths/sumset.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    >>> sumset({1, 2, 3}, {4, 5, 6, 7})
+    {5, 6, 7, 8, 9, 10}
+
```
Comment (CaedenPH) [inline maths/sumset.py:22]:
```suggestion
```

**Review by CaedenPH — APPROVED** [review]
@cclauss Looks good -

**Review by cclauss — APPROVED** [review]
RSLGTM

**Pull request merged**